pkg/router/template/router.go

```diff
@@ -428,9 +464,19 @@ func (r *templateRouter) writeDefaultCert() error {
 // watchMutualTLSCert watches the directory containing the certificates for
 // mutual TLS and reloads the router if the directory contents change.
 func (r *templateRouter) watchMutualTLSCert() error {
+	var caUpdateChannel chan struct{}
 	caPath := os.Getenv("ROUTER_MUTUAL_TLS_AUTH_CA")
+	// TODO: Determine the best place to put CRLs
+	crlPath := "/crls/crls.pem"
```

Comment:
Any reason not to add a subdirectory under `/var/lib/haproxy/router/`?  (Note that you would need to add such a subdirectory to the `mkdir` command in `Dockerfile.rhel8`.)  

I guess that might cause must-gather to include CRLs, which can be large, but it's probably worth that tradeoff if it makes diagnosing CRL issues easier.

Reply:
Correction: You don't need to `mkdir` the subdirectory if it comes from a volume mount.

Reply:
Which approach do you think is better? It sounds like creating the crl directory with `mkdir` is probably the more straightforward approach, plus it cuts down on the changes necessary to the ingress-operator since no `EmptyDir` volume mount would be necessary.